U4 4410

crgrieve opened the pull request:
Fixing: http://issues.umbraco.org/issue/U4-4410

Fixes MNTP comma separated allowed types when there is a space in the comma separated values.

clausjensen commented:
Hi @crgrieve 

I'm a bit curious as to why you reverted the fix done to `src/Umbraco.Web.UI.Client/src/views/common/dialogs/treepicker.controller.js` in your first commit, in the second commit?

Best,
Claus

crgrieve commented:
I realised I had changed it in the wrong place and only knew the scenario how to test /overlays/treepicker/treepicker.controller.js rather than /dialogs/treepicker.controller.js. So I reverted it as I didn't know where it's used to confirm the fix?

I assume it would work the same so should probably be in both places? If so, I can add it in again.

clausjensen commented:
Ahh I see :) fix is good for both scenarios - I've added it back in and merged your code!

Thanks for the PR!

https://github.com/umbraco/Umbraco-CMS/commit/cdb0c57416cc57679b25bf437a22c561e19c1804
